job/pytorchtrainingjob/ps/job.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: PyTorchTrainingJob
+metadata:
+  name: torch-mnist-trainingjob-ps
+spec:
+  scheduler:
```

Comment:
let's make this set of samples simple, and refrain from using `spec.scheduler` for now